Project-Aurora/Project-Aurora/Settings/Configuration.cs

```diff
@@ -8,7 +8,7 @@
 using Aurora.Profiles.Generic_Application;
 using Aurora.Profiles;
 using Newtonsoft.Json.Serialization;
-
+using keyboard;
```

Comment:
what is this using for?

Reply:
Thanks for reminding me. I am going to remove it.